### DISCUSSION TITLE
Use variable values in tooltips

### DISCUSSION THREAD

**Discussion by PabloPomodoro** [saadeghi/daisyui#2935] [Ideas / feature requests]
Hi,

as seemingly my reply in [this](https://github.com/saadeghi/daisyui/discussions/2907) thread went unnoticed, here my follow up questions:

Is it planned (or already possible) to enable the functionality, that one can use string variables in tooltips?
E. g. `data-tip="{{ stringVariable }}"` ?

Thank you!

**Comment by saadeghi** [comment]
daisyUI is just the CSS file. No compiling, data binding , HTML process or variable validation happens on our side. daisyUI only adds some CSS styles to your CSS file.  

For tooltip we simply show the value of `data-tip` in the CSS `:before` pseudo element.  

As I answered before, you should ask this question in Transloco repo because the error you're getting is about data binding (which is not related to daisyUI)  

daisyUI is just CSS. Even if you uninstall daisyUI dependency completely you will get the same error from Angular/Transloco.
